### ISSUE TITLE
Installed following steps. Top right icon still grayed out, does not show options/buffer.

### ISSUE THREAD

**Issue by Tarakanza** [YePpHa/YouTubeCenter#2464]
I installed the extension through policy changer.
It "works" as it is not grayed out in the list of extensions.

The topright icon is still grayed out.
Chrome redirects me when I select and click on "options". I cannot get to options.
Youtube still doesn't buffer.

Installed both:
YouTube Center 2.1.4
ID: bcegdpionpopahcglnfiiioapcclamdj

This product makes the whole youtube page go white, while playing only sound.
 YouTube Center Developer Build 548
ID: ajijnmbjgaeekdpmpohgppkckmnagimk

Enabled them, one at a time.

I cannot get to options. What should I do? Is it part of the google megacapitalist plot or is a bug in the plugin/extension/addon?

Thank you.

**Comment by FartingQuestion** [comment]
Go back to old layout, doesn't work on the new material layout.

**Comment by Tarakanza** [comment]
What is layout? Where do i change it? in youtube or in Chrome?

**Comment by FartingQuestion** [comment]
> To exit the Material Design layout, you can go to YouTube's homepage, click your channel icon in the top right to pop-up a menu, then click Restore classic YouTube at the bottom of its list.

**Comment by Tarakanza** [comment]
Everything works. Plug in doesn't want to update. "Error connecting to server"

How nice would it be if this plugin was like a crack for liberating software from capitalists, where all the policy changes, youtube view changes and other necessary adjustments were done by the plugin during installation.

Otherwise this great plugin is not becoming popular because it is very hard to install and there is no full installation list.

I had to:
Follow github steps, install policy changer then change the policy to allow chrome to use this plug in, then change youtube.

How nice if all those steps were written on the same page.

**Comment by Tarakanza** [comment]
Any way to trick youtube into buffering a full video in HD?

I would do it by sending requests to buffer the video at different places and than joining all the pieces.

As many poor people living in house shares with slow internet, I would appreciate such accelerator algorithm.

**Comment by halcion** [comment]
agreed, youtube center is in desperate need of an update, disable DASH playback no locnger works along with a laundry list of other features not working as of some weeks ago. already using old layout, that doesnt fix the issues
